examples/USExtractExample.php

```diff
@@ -35,9 +35,17 @@ public function run() {
         $lookup->isAggressive();
         $lookup->setAddressesHaveLineBreaks(false);
         $lookup->setAddressesPerLine(2);
+        try {
+            $client->sendLookup($lookup);
+            $this->displayResults($lookup);
+        }
+        catch (\Exception $ex) {
+            echo($ex->getMessage());
+        }
+    }
 
-        $client->sendLookup($lookup);
-
+    public function displayResults(Lookup $lookup)
```

Comment:
Is there a reason this one is public and the one in USAutocompleteProExample is private?